clickplc/tests/test_driver.py

```diff
@@ -145,6 +171,35 @@ async def test_dh_roundtrip(plc_driver):
     await plc_driver.set('dh500', 500)
     assert await plc_driver.get('dh500') == 500
 
+@pytest.mark.asyncio(loop_scope='session')
+async def test_sd_roundtrip(plc_driver):
+    """Confirm writable SD ints are read back correctly after being set."""
```

Comment:
Let's split out the roundtrips for `sd29` and `sd35` into a separate test that fully sets the time, since that's what you actually care about.

At some point I expect there will be explicit functions for each of the 'special' SD registers.

Reply:
I took my script and put it into two tests. See if that resolves what you're looking for.

Reply:
Looks great!